internal/examples/proto/gen/json/einride_bigquery_public_v1_dogecointransaction.json

```diff
@@ -2,149 +2,178 @@
   {
     "name": "hash",
     "type": "STRING",
-    "mode": "NULLABLE"
+    "mode": "NULLABLE",
+    "description": "STRING REQUIRED"
   },
   {
     "name": "size",
     "type": "INTEGER",
-    "mode": "NULLABLE"
+    "mode": "NULLABLE",
+    "description": "INTEGER NULLABLE"
   },
   {
     "name": "virtual_size",
     "type": "INTEGER",
-    "mode": "NULLABLE"
+    "mode": "NULLABLE",
+    "description": "INTEGER NULLABLE"
   },
   {
     "name": "version",
     "type": "INTEGER",
-    "mode": "NULLABLE"
+    "mode": "NULLABLE",
+    "description": "INTEGER NULLABLE"
   },
   {
     "name": "lock_time",
     "type": "INTEGER",
-    "mode": "NULLABLE"
+    "mode": "NULLABLE",
+    "description": "INTEGER NULLABLE"
   },
   {
     "name": "block_hash",
     "type": "STRING",
-    "mode": "NULLABLE"
+    "mode": "NULLABLE",
+    "description": "STRING REQUIRED"
   },
   {
     "name": "block_number",
     "type": "INTEGER",
-    "mode": "NULLABLE"
+    "mode": "NULLABLE",
+    "description": "INTEGER REQUIRED"
   },
   {
     "name": "block_timestamp",
     "type": "TIMESTAMP",
-    "mode": "NULLABLE"
+    "mode": "NULLABLE",
+    "description": "TIMESTAMP REQUIRED"
   },
   {
     "name": "block_timestamp_month",
     "type": "DATE",
-    "mode": "NULLABLE"
+    "mode": "NULLABLE",
+    "description": "DATE REQUIRED"
   },
   {
     "name": "input_count",
     "type": "INTEGER",
-    "mode": "NULLABLE"
+    "mode": "NULLABLE",
+    "description": "INTEGER NULLABLE"
   },
   {
     "name": "output_count",
     "type": "INTEGER",
-    "mode": "NULLABLE"
+    "mode": "NULLABLE",
+    "description": "INTEGER NULLABLE"
   },
   {
     "name": "is_coinbase",
     "type": "BOOLEAN",
-    "mode": "NULLABLE"
+    "mode": "NULLABLE",
+    "description": "TOOD: Figure out if there's a common protobuf type for NUMERIC.\nstring input_value = 12; // NUMERIC NULLABLE\nstring output_value = 13; // NUMERIC NULLABLE\nBOOLEAN NULLABLE"
```

Comment:
I think it would be nice to not have this comment generated as an example. So either if we configure this file to not output comments, or if we add some kind of ignore string to comments so that they are not generated as BigQuery field descriptions.

Reply:
It will be hard to make some general logic around it. I would argue that it's bad practice to commit commented out code anyway.

Reply:
Will the commented out code be removed from the description if we add a line break [here](https://github.com/einride/protobuf-bigquery-go/blob/aa6cd0f4917d2b5321e8b12f183eeb5e16196e41/internal/examples/proto/einride/bigquery/public/v1/dogecoin_transaction.proto#L24)? What do you think?